### DISCUSSION TITLE
Why is Run button disabled?

### DISCUSSION THREAD

**Discussion by numeralnathan** [erosman/support#452] [Q&A]
I have several scripts that run perfectly.  I created a new script that matches a page.  However, it won't run.  The manual "Run" button is disabled.  How do I figure out why the Run button is disabled?

**Comment by erosman** [comment]
Run button is disabled when the script is already injected into that page via automatic injection.

You should see that the script is active.

**Reply by numeralnathan** [reply]
How do I see that the script is active?  Do you mean that it appears in the Tabs & Scripts section of in the Fire Monkey drop down?

**Comment by erosman** [comment]
It would appear in **Tabs Scripts & CSS** with a :heavy_check_mark:  next to it to show that it is **enabled** and **active** on that page.
